### ISSUE TITLE
Not logging in again

### ISSUE THREAD

**Issue by dsgrau** [thiggy01/change-gdm-background#10]
When I run the command sudo ./ubuntu-20.04-change-gdm-background /path/to/image (I put the path to the image I want) it asks me to restart gdm, and when I put yes, it restarts with the new background image but when I try to log in, it loops on the login page. So I have to restart the computer and then the new background image is not set anymore.

Ubuntu 20.04 LTS 64-bit
GNOME Version 3.36.2

libglib2.0-dev-bin is already the newest version (2.64.2-1~fakesync1).

Whe I run chmod +x gdm-change-login-background it says "chmod: cannot access 'gdm-change-login-background': No such file or directory" but I supposed that you wanted to run "chmod +x ubuntu-20.04-change-gdm-background"

Any idea why is the problem that it loops on the login screen without letting me log in again?

**Comment by thiggy01** [comment]
This script just works with the default Ubuntu Yaru theme which is inside the gnome-shell-theme.gresource file.
You can see what's your current theme with the `update-alternatives --config gdm3-theme.gresource` command which has an asterisk at the beginning of the line containing the selected theme.
You can restore anytime to your previous theme with the `sudo ./ubuntu-20.04.change-gdm-background --restore` command.

**Comment by grandvizier** [comment]
:thinking: hmm, I had the same issue, and went with the rest but still not working for me
`update-alternatives --config gdm3-theme.gresource`
```
There are 2 choices for the alternative gdm3-theme.gresource (providing /usr/share/gnome-shell/gdm3-theme.gresource).

  Selection    Path                                                           Priority   Status
------------------------------------------------------------
* 0            /usr/share/gnome-shell/theme/Yaru/gnome-shell-theme.gresource   15        auto mode
  1            /usr/share/gnome-shell/gnome-shell-theme.gresource              10        manual mode
  2            /usr/share/gnome-shell/theme/Yaru/gnome-shell-theme.gresource   15        manual mode
```
is it maybe because there's a manual AND auto theme for Yaru?

**Comment by thiggy01** [comment]
@grandvizier that's the default configuration. Auto mode simply means that the system selected the theme with highest priority which Yaru theme, but If gnome theme was let's say priority 20 it would be automatically selected and it would be shown on the first row.
Can you provide more details? Can you login again after you restarted your computer? Can you see the background image you selected when you type y? There's any other output from the script other than saying it was successful and asking to restart gdm?

**Comment by grandvizier** [comment]
> Can you login again after you restarted your computer? 
   
yes, on restart it reverts back to the gorilla (regardless of which theme is selected)

> Can you see the background image you selected when you type y? 
   
yes, it loads up the login prompt with the new background image, but when I try to log in, it freezes with only showing the new image. 

> There's any other output from the script other than saying it was successful and asking to restart gdm?

This is the extent of the output:
```
$ sudo ./change-gdm-background /usr/share/backgrounds/Camera_Film_by_Markus_Spiske.jpg
test
GDM background sucessfully changed.
Do you want to restart gdm to apply change? (y/n):
```

I've also attempted selecting `n` and then restarting the gdm service myself, but that results the same freezing. I've also attempted selecting `n` and restarting, but that also just reverts back to the gorilla. 

---- 
another thing to note, when selecting `y` and the new image loads, the theme changes from the "default gorilla" that I'm used to. I can't manage a screenshot when not logged in, but i found example images to show what i mean.

default ![default gorilla](https://i.stack.imgur.com/mYwdX.jpg)

new ![new background ](https://lh3.googleusercontent.com/-U_VjUGPUMqI/X5Bamqoh6CI/AAAAAAAAdys/_pMhHmLeDlMf5jlTnACNWUCHb_4nqjpdgCLcBGAsYHQ/groovy-starts-031.png)
